server/data/activitiesApiClient.ts

```diff
@@ -302,6 +303,13 @@ export default class ActivitiesApiClient extends AbstractHmppsRestClient {
     })
   }
 
+  async getAppointmentLocations(prisonCode: string, user: ServiceUser): Promise<AppointmentLocationSummary[]> {
```

Comment:
While I'm not certain of the value of it, there are tests for these client and service methods. Perhaps add them to maintain that pattern/coverage?

Reply:
test added